src/main/java/edu/ie3/datamodel/io/factory/input/SwitchInputFactory.java

```diff
@@ -35,16 +35,14 @@ protected SwitchInput buildModel(
       OperationTime operationTime) {
     final boolean closed = data.getBoolean(CLOSED);
 
-    try {
-      final int parallelDevices = data.getInt(PARALLEL_DEVICES);
+    if (data.containsKey(PARALLEL_DEVICES)) {
+      String parallelDevices = data.getField(PARALLEL_DEVICES);
+
       log.warn(
-          "Found a `SwitchInput` with the id `{}` that specifies `parallelDevices` with a value of `{}`. Because switches cannot be parallel, the given value is ignored!",
+          "The `SwitchInput` with the id `{}` specifies the unsupported parameter `parallelDevices` with a value of `{}`."
+              + " Because switches are expected to have a given value `1`, the provided value is ignored.",
```

Comment:
Hmmm, the implicit value is not really 1, is it? I guess it really depends on the context. Maybe something like `SwitchInputs do not need to specify parallelDevices, as its value depends on the electrotechnical context where the switch is embedded.`